### PR TITLE
docs(docs-infra): reformat redundant sentence

### LINKED ISSUE #38106: redundant reference to npm
# 📚 Docs or angular.io bug report

### Description

On page [Using published libraries](https://github.com/angular/angular/blob/deac9945176f2668752b479450ce4859d7977d72/aio/content/guide/using-libraries.md) line 11 is quoted

> The ng add command uses the npm package manager or yarn (...)

Considering that npm is literally node package manager, this sentence sounds redundant

### PR DESCRIPTION
**Pull request by saifabusaleh**
reformat sentence uses the npm package manager  since npm is node package manager

Fixes #38106

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

### CODE REVIEW

File: aio/content/guide/using-libraries.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ See the [Angular Resources](resources) page for links to the most popular ones.
 Libraries are published as [npm packages](guide/npm-packages), usually together with schematics that integrate them with the Angular CLI.
 To integrate reusable library code into an application, you need to install the package and import the provided functionality where you will use it. For most published Angular libraries, you can use the Angular CLI `ng add <lib_name>` command.
 
-The `ng add` command uses the npm package manager or [yarn](https://yarnpkg.com/) to install the library package, and invokes schematics that are included in the package to other scaffolding within the project code, such as adding import statements, fonts, themes, and so on.
+The `ng add` command uses node package manager such as [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) to install the library package, and invokes schematics that are included in the package to other scaffolding within the project code, such as adding import statements, fonts, themes, and so on.
```
Comment (petebacondarwin) [inline aio/content/guide/using-libraries.md:11]:
```suggestion
The `ng add` command uses a package manager such as [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) to install the library package, and invokes schematics that are included in the package to other scaffolding within the project code, such as adding import statements, fonts, themes, and so on.
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
I think `node` is also redundant here, and the article makes it read better.

File: aio/content/guide/using-libraries.md (hunk 1 above)
Comment (saifabusaleh) [reply aio/content/guide/using-libraries.md]: @petebacondarwin Updated

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

Great! Thanks. Please can you either squash the two commits into one, or change the second commit to a "fixup" commit, and then repush?

**Comment by saifabusaleh** [comment]
Not sure that the squash went well 

I think I did merge instead..

**Comment by saifabusaleh** [comment]
@petebacondarwin Took me a while to do this squash correctly..

Should be ok now :) 

Thanks

**Comment by petebacondarwin** [comment]
Great stuff! Thanks.

**Pull request closed without merging**